packages/node_modules/@webex/plugin-messages/test/integration/spec/messages.js

```diff
@@ -377,6 +377,79 @@ describe('plugin-messages', function () {
           });
       });
     });
+
+    describe('when a message is threaded', () => {
```

Comment:
Almost there! Would like to see this within the `#list` describe block as well, so this whole describe block added after line 378.

Reply:
@adamweeks done.